code/modules/mob/say.dm

```diff
@@ -1,7 +1,45 @@
-/mob/verb/say_verb(message as text)
+/mob/verb/say_window_ok()
+	set name = ".say_window_ok"
+	set hidden = TRUE
+
+	say_verb(winget(client, "say_window.say_input", "text"))
+
+/mob/verb/say_window_close()
+	set name = ".say_window_close"
+	set hidden = TRUE
+
+	winset(client, "say_window", "is-visible=false")
+	winset(client, "say_window.say_input", "text=")
+
+/mob/verb/say_window_open()
```

Comment:
Maybe I am a little paranoid, but it worth checking if `usr == src` here ? Can you fuck with winsets by typing `byond://[url]` ?

Reply:
This is a verb and wouldn't be the relevant place to put such a check